Gemfile.lock

```diff
@@ -3,7 +3,7 @@ GEM
   specs:
     CFPropertyList (3.0.4)
       rexml
-    activesupport (5.2.6)
+    activesupport (5.2.6.2)
```

Comment:
Off topic: I'd never seen this kind of versioning 🤔 But Rails has been using it for quite some time, apparently:

<img alt="2022-02-18 12-59-39 2022-02-18 13_00_36" src="https://user-images.githubusercontent.com/1218433/154603359-22b1d339-ef77-4fed-8534-23a36249cbb6.gif" width=300/>